.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "python.testing.pytestArgs": [
+        "tests"
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true
+}
```

Comment:
@berrydenhartog we could also add a setting to autoformat and sort based on Ruff. What do you think?

See: https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff#configuring-vs-code

In a recent release note, Ruff also mentions something about a built-in language server: https://marketplace.visualstudio.com/items?itemName=charliermarsh.ruff#enabling-the-rust-based-language-server

Reply:
This is already set in the devcontainer ;)

Reply:
see: https://github.com/MinBZK/tad/blob/9ff593b10841af017d78c3879834a932d7863d15/.devcontainer/devcontainer.json#L45